Frontend/ui-customer/src/Flow.purs

```diff
@@ -309,20 +309,20 @@ currentFlowStatus = do
         else updateFlowStatus SEARCH_CANCELLED
 
 
-chooseLanguageScreenFlow :: FlowBT String Unit
-chooseLanguageScreenFlow = do
-  logField_ <- lift $ lift $ getLogFields
-  hideLoaderFlow
-  setValueToLocalStore LANGUAGE_KEY $ getValueFromConfig "defaultLanguage"
-  _ <- lift $ lift $ liftFlow $ logEvent logField_ "ny_user_choose_lang_scn_view"
-  flow <- UI.chooseLanguageScreen
-  case flow of
-    NextScreen language -> do
-                            setValueToLocalStore LANGUAGE_KEY language
-                            void $ pure $ setCleverTapUserProp [{key : "Preferred Language", value : unsafeToForeign language}]
-                            _ <- lift $ lift $ liftFlow $(logEventWithParams logField_ "ny_user_lang_choose" "language" (language))
-                            enterMobileNumberScreenFlow
-    Refresh state -> chooseLanguageScreenFlow
+-- chooseLanguageScreenFlow :: FlowBT String Unit
```

Comment:
why is this commented? please add proper comments for it (if needed to be commented)